index.html

```diff
@@ -216,11 +225,15 @@ <h5 class="text-white">Skills</h5>
             </div>
         </section>
       <form action="https://formspree.io/f/xlezlkyw" method="post">
-        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="forms">  
+        <p class="mine">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="inputs">
```

Comment:
- [ ] Kindly style the form to match the design template please.
## Your design
![r3](https://user-images.githubusercontent.com/94551484/186989355-40ef3804-9c72-4a48-be4a-36c56cc570be.png)
## Figma template
![s3](https://user-images.githubusercontent.com/94551484/186989371-4ad8bbc7-b9be-48a5-b62a-3a836ab5bc35.png)

Reply:
I kindly advise you to follow the previous reviewer's advice.